### ISSUE TITLE
TwoHandManipulatable script moves objects even if OneHandMovement option is unchecked

### ISSUE THREAD

**Issue by Baxab** [Microsoft/MixedRealityToolkit-Unity#2021]
## Overview
I hope this is not a duplicate. I checked and it seems that nobody has reported this behaviour yet.
As you can get from the title, there is a bug in TwoHandManipulatable script. It's possible to move objects using one hand even if OneHandMovement option is unchecked.

## Expected Behavior
While manipulating objects in one of TwoHand state (Scaling, Rotating, MovingScaling, RotatingScaling, MovingRotatingScaling), if I lose track of one of my hands I expect the action (move, scale, rotate) that I was doing to stop.

## Actual Behavior
However, if I lose track of one of my hands the script automatically enters into the Moving state, allowing me to move the object I was manipulating with one hand.

## How I solved it
Basically, I sorted out the problem by changing line 290 of TwoHandManipulatable script.
From:
newState = State.Moving;
To:
newState = State.Start;

Now everything seems to work as expected.

## Unity Editor Version
Unity 2017.2.0f3
## Mixed Reality Toolkit Release Version
2017.2.1.4 Patch Release

**Comment by keveleigh** [comment]
This should be fixed via #2459. Please take a look at the June branch to verify! Thanks!
